### PR TITLE
Feat : update function

### PR DESCRIPTION
**Pull request by sukyungdev**
Add update(modify) note function

### CODE REVIEW

**Review by ha-anna — APPROVED** [review]

**Review by genelorenzSarmiento0408 — APPROVED** [review]
LGTM!

File: main.js [hunk 1]
```diff
@@ -98,15 +102,91 @@ function deleteNote(e) {
...
+			console.log(notes[i]);
+
+			localStorage.setItem('updateNoteData', JSON.stringify(notes[i]));
+			window.location.href = './update.html';
+			break;
+		}
+	}
+}
+
+updateRenderNote();
```
Comment (genelorenzSarmiento0408) [inline main.js:138]: What is this for?

**Review by genelorenzSarmiento0408 — APPROVED** [review]
LGTM!

File: main.js (hunk 1 above)
Comment (sukyungdev) [reply main.js:138]: > What is this for?
It is a code to call updateRenderNote();.
updateRenderNote() were not worked in other functions 🤔

**Pull request merged**